src/types/incident.ts

```diff
@@ -21,15 +21,12 @@ type ExtraProps = {
 export interface Incident extends Record<string, any>, ExtraProps {
   category: string
   classification: Classification | null
-  datetime: Datetime
+  dateTime: number | null
```

Comment:
why is this a number and not a datetime type?

Reply:
Because `Datetime` type is an object that is constructed to meet the data structure that is used in radio button list components. Using an object to determine something as basic as a point in time is superfluous. Timestamps can be used to calculate diffs and can be formatted to readable strings.